app/models/article.rb

```diff
@@ -243,12 +242,22 @@ def self.active_eli5(time_ago)
     stories.pluck(:path, :title, :comments_count, :created_at)
   end
 
-  def body_text
-    ActionView::Base.full_sanitizer.sanitize(processed_html)[0..7000]
-  end
+  def self.seo_boostable(tag = nil, time_ago = 18.days.ago)
+    time_ago = 5.days.ago if time_ago == "latest" # Time ago sometimes returns this phrase instead of a date
+    time_ago = 75.days.ago if time_ago.nil? # Time ago sometimes is given as nil and should then be the default. I know, sloppy.
 
-  def index_id
-    "articles-#{id}"
+    relation = Article.published.
+      order(organic_page_views_past_month_count: :desc).
+      where("score > ?", 8).
+      where("published_at > ?", time_ago).
+      limit(25)
+
+    fields = %i[path title comments_count created_at]
+    if tag
+      relation.cached_tagged_with(tag).pluck(*fields)
+    else
+      relation.pluck(*fields)
+    end
```

Comment:
refactored this bit. see https://github.com/thepracticaldev/dev.to/pull/4882/files#diff-9b9a42a96369816cd75b7c8b440538a8L397